sotodlib/coords/demod.py

```diff
@@ -180,4 +180,21 @@ def from_map(tod, signal_map, cuts=None, flip_gamma=True, wrap=False, modulated=
         if wrap:
             tod.wrap('signal', signal_sim, [(0, 'dets'), (1, 'samps')])
         return signal_sim
-    
\ No newline at end of file
+
+
+def rotate_demodQU(tod, zero_gamma=True):
+    """
+    Apply detectors' polarization angle calibration to the HWP demodulated Q and U timestreams.
```

Comment:
I think worth stating that this puts all detectors in a common Q/U relative to the instrument frame.